py/desitarget/mock/build.py

```diff
@@ -855,47 +855,40 @@ def _update_spectra_status(result):
                 log.info('No {} standards on healpix {}, {} not written.'.format(stdsuffix, pixnum, stdfile))
 
         # Finally write out the rest of the targets.
-        # targetsfile = os.path.join(output_dir, 'targets-{}.fits'.format(healsuffix))
-        # truthfile = os.path.join(output_dir, 'truth-{}.fits'.format(healsuffix))
-        # truthspecfile = os.path.join(output_dir, 'spectra-truth-{}.fits'.format(healsuffix))
-
         targetsfile = mockio.findfile('targets', nside, pixnum, basedir=output_dir)
         truthfile = mockio.findfile('truth', nside, pixnum, basedir=output_dir)
```

Comment:
Doesn't `desispec.io.util.findfile` have the functionality to find these files, given a different top-level path?